### DISCUSSION TITLE
multiple box shadows

### DISCUSSION THREAD

**Discussion by neatlittlemacapps** [six7/figma-tokens#231] [Q&A]
Currently, defining a box shadow can only hold 1 set of values. Whilst in reality (and figma can handle this too through their effects functionality) you can "stack" multiple sets in 1 box shadow. Is there some way of obtaining the same thing with this plugin?

Something like this:  
box-shadow: 0 3px 4px 0 rgba(0, 0, 0, 0.20), 0 3px 3px -2px rgba(0, 0, 0, 0.14), 0 1px 8px 0 rgba(0, 0, 0, 0.14);

**Comment by six7** [comment]
Yes, this is what I would ultimately love for box shadows. Right now it's an object, but I think really we should store it as a string or an array, e.g. `[0 3px 4px 0 rgba(0, 0, 0, 0.20), 0 3px 3px -2px rgba(0, 0, 0, 0.14), 0 1px 8px 0 rgba(0, 0, 0, 0.14)]`

I'll create an issue for that.

**Reply by neatlittlemacapps** [reply]
U so rock... Thx!

**Reply by neatlittlemacapps** [reply]
Oh, one more thing (deng I wish I could code): Could you also add a "description" field to box shadows? It would allow me to add usage descriptions.

**Reply by six7** [reply]
Definitely, will add that to a quickfix list. This should definitely be in.

**Comment by sschiffer** [comment]
Just started to use figma tokens. Would love to have this in place!

**Reply by six7** [reply]
This feature is already in the plugin! Just click the + when creating/editing a shadow token.

**Reply by van-nguyen-ht** [reply]
hello Jan, can I write this for a card dropshadow token= {dropshadow}+{dropshadow.light}?
